templates/livecoms-template-bestpractices.tex

```diff
@@ -243,12 +243,46 @@ \subsection{Lists}
 \item[Idea] Text
 \end{description}
 
-Some filler text, because empty templates look really poorly. \lipsum[1]
-
-
-\section{Acknowledgments}
-
-Funder and other information can be given here.
+Some filler text, because empty templates look really poor. \lipsum[1]
+
+\section{Author Contributions}
+%%%%%%%%%%%%%%%%
+% Attribute authors into the general categories:
+% * ''wrote sections X and Y''.
+% * ''contributed to all sections''.
+% * ''contributed to section X.''
+% * ''performed and analyzed the experiments in section Y''.
+% See the policies ``Policies on Authorship'' section of https://livecoms.github.io for more information on
+% deciding on authorship and author order.
+%%%%%%%%%%%%%%%%
+FMS wrote sections X and Z. RMS contributed to the writing of sections Y and W.
+TMP contributed to the writing of sections Y and W. ITP contributed to all sections.
+For a more detailed description of contributions,
+see the GitHub issue tracking and changelog at \url{\githubrepository}.
+
+\section{Other Contributions}
+%%%%%%%%%%%%%%%
+% You should include all people who have filed issues that were
+% accepted into the paper, or that upon discussion altered what was in the paper.
+% Multiple significant contributions might mean that the contributor
+% should be moved to authorship at the discretion of the a
+% See the policies ``Policies on Authorship'' section of https://livecoms.github.io for
+%more information on deciding on authorship and author order.
+%%%%%%%%%%%%%%%
+EGP provided inportant insights on the challenges faced by nonequilibrium simulations.
+FTP provided several key citations on energy conservation. For a more detailed description of contributions,
+see the GitHub issue tracking and changelog at \url{\githubrepository}.
```

Comment:
Can the url command be embedded in the macro?

Reply:
I was thinking that one might want to use it in other contexts, but perhaps not.  I'll see if it nests.

Reply:
Looks like it can be done.